### PR TITLE
Use more lines to check for prompt at the end of the output

### PR DESCRIPTION
**Pull request by sriramkannan95**
In our project our Bash prompt is modded to look similar to ksh like:

> hostname:/current/directory
> $

So the prompt makes up last two lines. This enhancement will allow more than just the last line while looking for the prompt.

### CODE REVIEW

**Comment by sriramkannan95** [comment]
@fgimian
@fruch

Can you please review the changes, I have tested them with my current networking infra with bash and ash

**Comment by fruch** [comment]
I'm kind of blind without the tests, let me see when I'll time to fix them.

File: paramiko_expect.py [hunk 1]
```diff
@@ -56,12 +56,14 @@ class SSHClientInteraction(object):
...
     :param encoding: The character encoding to use.
+    :param lines_to_check: The number of last few lines of the output to
+                           look at, while matching regular expression(s)
     """
 
     def __init__(
         self, client, timeout=60, newline='\r', buffer_size=1024,
         display=False, encoding='utf-8', output_callback=default_output_func,
-        tty_width=80, tty_height=24
+        tty_width=80, tty_height=24, lines_to_check=2
```
Comment (fgimian) [inline paramiko_expect.py:66]: This should probably default to `1` to avoid breaking backwards compatibility.

**Comment by fgimian** [comment]
> @fgimian @fruch
> 
> Can you please review the changes, I have tested them with my current networking infra with bash and ksh

While I wrote this library, I have not used it in years so I'm not really the primary maintainer anymore.  @fruch thanks heaps for helping out.

Cheers
Fotis

File: paramiko_expect.py (hunk 1 above)
Comment (sriramkannan95) [reply paramiko_expect.py]: Done. Thanks.

**Comment by sriramkannan95** [comment]
> I'm kind of blind without the tests, let me see when I'll time to fix them.

@fruch I am able to run the tests locally. Will attempt to fix the automated tests

---------- coverage: platform darwin, python 3.9.7-final-0 -----------
Name                                                           Stmts   Miss  Cover   Missing
--------------------------------------------------------------------------------------------
/opt/homebrew/lib/python3.9/site-packages/paramiko_expect.py     152      3    98%   181, 235, 243
--------------------------------------------------------------------------------------------
TOTAL                                                            152      3    98%

=============================================================== 16 passed, 1 warning in 22.54s ===============================================================                     

https://privatebin.at/?0939064d8e715337#GydABc8RPznWM4G5dr88EZHpuNiy1RPnLKDWJhJfxqNu

**Pull request merged**